addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/EnOceanBindingConstants.java

```diff
@@ -0,0 +1,295 @@
+    public final static String CHANNEL_DIMMER = "dimmer";
+    public final static String CHANNEL_ROLLERSHUTTER = "rollershutter";
+    public final static String CHANNEL_ANGLE = "angle";
+    public final static String CHANNEL_TEMPERATURE = "temperature";
+    public final static String CHANNEL_HUMIDITY = "humidity";
+    public final static String CHANNEL_SETPOINT = "setPoint";
+    public final static String CHANNEL_FANSPEEDSTAGE = "fanSpeedStage";
+    public final static String CHANNEL_OCCUPANCY = "occupancy";
+    public final static String CHANNEL_MOTIONDETECTION = "motionDetection";
+    public final static String CHANNEL_ILLUMINATION = "illumination";
+    public final static String CHANNEL_COUNTER = "counter";
+    public final static String CHANNEL_CURRENTNUMBER = "currentNumber";
+
+    public final static String CHANNEL_PUSHBUTTON = "pushButton";
+
+    public final static String CHANNEL_ROCKERSWITCH_CHANNELA = "rockerswitchA";
+    public final static String CHANNEL_ROCKERSWITCH_CHANNELB = "rockerswitchB";
+
+    public final static String CHANNEL_VIRTUALROCKERSWITCHA = "virtualRockerswitchA";
+    public final static String CHANNEL_VIRTUALROCKERSWITCHB = "virtualRockerswitchB";
+    public final static String CHANNEL_ROCKERSWITCHLISTENERSWITCH = "rockerswitchListenerSwitch";
+    public final static String CHANNEL_ROCKERSWITCHLISTENERROLLERSHUTTER = "rockerswitchListenerRollershutter";
+
+    public final static String CHANNEL_WINDOWHANDLESTATE = "windowHandleState";
+    public final static String CHANNEL_CONTACT = "contact";
+    public final static String CHANNEL_TEACHINCMD = "teachInCMD";
+    public final static String CHANNEL_INSTANTPOWER = "instantpower";
+    public final static String CHANNEL_TOTALUSAGE = "totalusage";
+    public final static String CHANNEL_INSTANTLITRE = "amrLitre";
+    public final static String CHANNEL_TOTALCUBICMETRE = "amrCubicMetre";
+
+    public final static String CHANNEL_AUTOOFF = "autoOFF";
+    public final static String CHANNEL_DELAYRADIOOFF = "delayRadioOFF";
+    public final static String CHANNEL_EXTERNALINTERFACEMODE = "externalInterfaceMode";
+    public final static String CHANNEL_TWOSTATESWITCH = "twoStateSwitch";
+    public final static String CHANNEL_ECOMODE = "ecoMode";
+
+    public final static String CHANNEL_RECEIVINGSTATE = "receivingState";
+
+    public final static String CHANNEL_GENERIC_SWITCH = "genericSwitch";
+    public final static String CHANNEL_GENERIC_ROLLERSHUTTER = "genericRollershutter";
+    public final static String CHANNEL_GENERIC_DIMMER = "genericDimmer";
+    public final static String CHANNEL_GENERIC_NUMBER = "genericNumber";
+    public final static String CHANNEL_GENERIC_STRING = "genericString";
+    public final static String CHANNEL_GENERIC_COLOR = "genericColor";
+    public final static String CHANNEL_GENERIC_TEACHINCMD = "genericTeachInCMD";
+
+    public final static ChannelTypeUID VirtualRockerSwitchAChannelType = new ChannelTypeUID(BINDING_ID,
```

Comment:
Please only use capitals for ALL constant names defined in this class.

Reply:
fixed